manifests/apache.pp

```diff
@@ -1,34 +1,118 @@
 # Configure an Apache vhost
 # @api private
-class pulpcore::apache {
+class pulpcore::apache (
+  String[1] $vhost_priority = '10',
```

Comment:
Is this something typically configured by a user?

Reply:
No, but it's a private class. There is 0 benefit to hardcoding it and only downsides if the user did modify it.

Reply:
As [discussed](https://github.com/theforeman/puppet-foreman_proxy_content/pull/271#issuecomment-654416991), it turns it this actually needs to be part of the public API.